tasks/prime.py

```diff
@@ -242,8 +244,10 @@ def on_start(self):
         email = json_resp["email"]
         user_id = json_resp["id"]
 
-        origin_duty_stations = self.client.get(self.customer_path("/internal/duty_stations?search=29"))
-        current_station_id = origin_duty_stations.json()[0]["id"]
+        # Setup customer profile
+        duty_stations = self.client.get(self.customer_path("/internal/duty_stations?search=palms"))
```

Comment:
Thanks for making this more explicit.  I do know the default GBLOC of the office users being created is now `KKFA`.  I think the origin duty station matters for the services counselor but not for the TOO.  I don't think Twenty Nine Palms provides services counseling so this move should proceed to the TOO queue.

We will want the shipment's postal code to map to the KKFA gbloc probably though so it appears in that TOO user's queue.